apps/redirect/web/.env.local.example

```
@@ -0,0 +1,39 @@
+# Integration tests provision an ISOLATED database (f3redirect_test) on this
+# same server via scripts/reset-test-db.mjs; point this at the server only.
+TEST_DATABASE_URL=postgresql://f3local:f3local@localhost:5433/f3nation_test
+
+# -- Better Auth (email+password primary, passkey secondary) --
+# Generate: openssl rand -base64 32
+BETTER_AUTH_SECRET=local-dev-secret-change-me-32-chars-min
+BETTER_AUTH_URL=http://localhost:3007
+
+# -- Passkey (WebAuthn) — defaults are fine for localhost --
+PASSKEY_RP_ID=localhost
+PASSKEY_RP_NAME=F3 Redirect
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)